### PR TITLE
[1LP][RFR] C&U: skipping for cloud provider

### PR DESCRIPTION
**Pull request by digitronik**
Purpose or Intent
=================
- https://bugzilla.redhat.com/show_bug.cgi?id=1525296
- block some tests related to raw_metric
- **Note: Collect for All Clusters must be checked to be able to collect C & U data from Cloud Providers such as Red Hat OpenStack or Amazon EC2**

{{pytest: cfme/tests/test_utilization_metrics.py -v --long-running }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 8ba82b6cca5e8bb73a21390d618fc6028404763f:

`cfme/tests/test_utilization_metrics.py`:
- :bangbang: Line 195:1: **E303** *too many blank lines (3)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by digitronik** [comment]
Disk and Network giving a problem. Vsphere also showing zero data. so best to discuss with @nachandr  . Remaining cloud test uncollected as per BZ.

File: cfme/tests/test_utilization_metrics.py [hunk 1]
```diff
@@ -136,8 +136,7 @@ def query_metric_db(appliance, provider, metric, vm_name=None, host_name=None):
 
 
 # Tests to check that specific metrics are being collected
-@pytest.mark.uncollectif(
-    lambda provider: current_version() < "5.7" and provider.type == 'gce')
+@pytest.mark.meta(blockers=[BZ(1525296, unblock=lambda provider: provider.category != 'cloud')])
```
Comment (izapolsk) [inline cfme/tests/test_utilization_metrics.py]: REQUIRED: please use provider.one_of(CloudProvider) instead

Comment (digitronik) [reply cfme/tests/test_utilization_metrics.py]: @izapolsk improve as you suggested.

**Pull request merged**